Moblin/View/Settings/RtmpServer/RtmpServerStreamSettingsView.swift

```diff
@@ -109,20 +109,68 @@ struct RtmpServerStreamSettingsView: View {
                     valueFormat: { "\($0) ms" }
                 )
                 .disabled(model.rtmpServerEnabled())
-                TextEditNavigationView(
-                    title: String(localized: "FPS"),
-                    value: String(stream.fps!),
-                    onSubmit: submitFps,
-                    footer: Text("""
-                    Force given FPS, or set to 0 to use the publisher's FPS. B-frames \
-                    does not work with forced FPS. Forced FPS is typically needed for DJI drones.
-                    """),
-                    keyboardType: .numbersAndPunctuation
-                )
-                .disabled(model.rtmpServerEnabled())
             } footer: {
                 Text("The stream name is shown in the list of cameras in scene settings.")
             }
+            Section {
+                Toggle("Bugged Publisher", isOn: Binding(get: {
+                    stream.buggedPublisher!
+                }, set: { value in
+                    stream.buggedPublisher = value
+                    if !value {
+                        stream.manualFps = false
+                    }
+                    model.store()
+                    model.reloadRtmpServer()
+                    model.objectWillChange.send()
+                }))
+                .disabled(model.rtmpServerEnabled())
+                if stream.buggedPublisher == true {
+                    Toggle("Manual FPS", isOn: Binding(get: {
+                        stream.manualFps!
+                    }, set: { value in
+                        stream.manualFps = value
+                        model.store()
+                        model.reloadRtmpServer()
+                        model.objectWillChange.send()
+                    }))
+                    .disabled(model.rtmpServerEnabled())
+                }
+                if stream.buggedPublisher == true, stream.manualFps == true {
+                    TextEditNavigationView(
+                        title: String(localized: "FPS"),
+                        value: String(stream.fps!),
+                        onSubmit: submitFps,
+                        keyboardType: .numbersAndPunctuation
+                    )
+                    .disabled(model.rtmpServerEnabled())
+                }
+            } footer: {
+                Text("""
+                Enable this setting to allow Moblin to estimate the FPS in the absence of accurate \
+                timestamps from the publisher. Alternatively, the FPS can be specified manually.
+                """)
+            }
+            Section {
+                Toggle("Auto set RTMP Scene", isOn: Binding(get: {
+                    stream.autoSelectRtmpScene!
+                }, set: { value in
+                    stream.autoSelectRtmpScene = value
+                    model.store()
+                    model.reloadRtmpServer()
+                    model.objectWillChange.send()
+                }))
```

Comment:
As no other camera is auto selected I think this one shouldn't be either. Have to keep it consistent.